aptos-move/mvhashmap/src/versioned_aggregators.rs

```diff
@@ -5,53 +5,60 @@ use crate::types::{AtomicTxnIndex, MVAggregatorsError, TxnIndex};
 use aptos_aggregator::{
     aggregator_change_set::{AggregatorApplyChange, ApplyBase},
     delta_change_set::DeltaOp,
-    types::{AggregatorID, AggregatorValue},
+    types::AggregatorValue,
 };
 use claims::{assert_matches, assert_none};
 use crossbeam::utils::CachePadded;
 use dashmap::DashMap;
 use std::{
     collections::btree_map::{BTreeMap, Entry},
+    fmt::Debug,
+    hash::Hash,
     iter::DoubleEndedIterator,
     sync::atomic::Ordering,
 };
 
+pub enum CommitError {
+    CodeInvariantError(String),
+    ReExecutionNeeded(String),
```

Comment:
I think this is / will be only used for delta application errors, so we shouldn't need a string? (plus we could have more specific error variants)

Reply:
I'm fine with that. I meant that we never print it and it seems it's always the same message that may not add much